bpf/bpf_hostdev_ingress.c

```diff
@@ -8,16 +8,24 @@
 #include <netdev_config.h>
 
 #include "lib/common.h"
+#include "lib/dbg.h"
 
 __section("to-host")
 int to_host(struct __ctx_buff *ctx)
 {
-	/* Upper 16 bits may carry proxy port number, clear it out */
-	__u32 magic = ctx_load_meta(ctx, 0) & 0xFFFF;
+	__u32 magic = ctx_load_meta(ctx, 0);
 
-	if (magic == MARK_MAGIC_TO_PROXY) {
+	if ((magic & MARK_MAGIC_HOST_MASK) == MARK_MAGIC_ENCRYPT) {
```

Comment:
@jrfastab would we be safe to also use `0xFFFF` to mask here? At the moment it's only masking on `0x0F00` so any `0xXEXX` value would match.

I refrained from making any changes like this in this PR since it's intended only as a cleanup, no functional changes.

Reply:
I think this should be OK I can't think of any reason why this wouldn't work.

Reply:
followup question for extra points, how confident are you that the CI would pick up a regression if I made this change? ;-)